### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by thoov**
@stefanpenner @blimmer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# ember-state-services Changelog
+
+### 3.1.0
+
+- [#51](https://github.com/stefanpenner/ember-state-services/pull/51) Use getOwner API to lookupFactory
+
+### 3.0.0
+
+- [#47](https://github.com/stefanpenner/ember-state-services/pull/47) Add default factory and improve initialState
```
Comment (stefanpenner) [inline CHANGELOG.md]: this should be labeled a [BREAKING] i think

**Comment by stefanpenner** [comment]
one nit-pick, then 👍

**Comment by thoov** [comment]
@stefanpenner Done

**Comment by stefanpenner** [comment]
@thoov awesome, thank you!

**Pull request merged**